test/manifolds/projective_space.jl

```diff
@@ -284,8 +272,8 @@ include("../utils.jl")
         end
 
         @testset "equivalence" begin
-            x = Quaternion[1.0 + 0im, 0.0, 0.0]
-            v = Quaternion[0.0, im, 0.0]
+            x = Quaternion[1.0, 0.0, 0.0]
+            v = Quaternion[0.0, 0.0, 0.0]
```

Comment:
Restores original point
```suggestion
            v = [0.0, quat(0, 1, 0, 0), 0.0]
```